README.md

```diff
@@ -1,20 +1,49 @@
-This repo contains a list of Engineering titles, roles, and associated responsibilities:
+The Mariana Tek Engineering Ladder defines the responsibilities and expectations of people working at various levels within the engineering team. It exists as a guide for engineers pursuing excellence and advancement, as well as a framework for managers to evaluate performance and identify candidates for promotion.
 
+Like many engineering teams, we outline two paths for career advancement: the management track and the technical track. Engineers pursuing the management track ("Managers") are interested in managing people and teams. Engineers pursuing the technical track ("Makers") are interested in honing their technical expertise and shaping the technical direction of the company.
+
+Framed another way by the engineering team at [Chartbeat](https://engineering.chartbeat.com/2015/06/05/engineering-ladders/), the difference between Makers and Managers is how they answer this question:
+
+> “Do I want to build bigger and better systems, or do I want to manage bigger and better teams?”
+
+Every engineer starts their career as a maker. As you work your way from Engineer I to Engineer II to Senior Engineer, you are primarily focused on expanding your technical knowledge and mastering the skills required of a professional software engineer. When you're ready to move beyond Senior Engineer, you have the opportunity to specialize.
```

Comment:
Confused by this portion because I am not senior but am specialized in front-end

Reply:
I will rephrase to clarify that I mean choosing between the technical track + managerial track. Good note!

Reply:
👍  This section is really well written and so much better than before.  @tracyalison11 makes a great call out about specialization though.  I think all of our engineers are somewhat specialized (mobile, frontend, backend, develops, ect...) and we seem to be trending towards more and more specialized teams.  If anything it almost seems like our expectations would be the reverse where as you become more and more senior you'd be expected to branch out and learn more about our entire stack vs the part you're initially comfortable in.  But tbh I'm not really sure what the expectation is there.